.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - run:
           name: Download dependencies
           command: |
-            git clone -b ${CI_BRANCH:-master} git@github.com:StackStorm-Exchange/ci.git ~/ci
+            git clone -b ${CI_BRANCH:-master} git@github.com:Anshika-Gautam/ci.git ~/ci
```

Comment:
Need to fix this

Reply:
Our pack is using mam-sdk which is dependent on iotfunctions package for its functioning. The "git@github.com:StackStorm-Exchange/ci.git ~/ci" is installing the pip version 9.0.3 which is not able to get the iotfunctions package from the specified git repository. It is resulting into the below mentioned error on circle:
**************************************************************************
`Using /home/circleci/virtualenv/lib/python3.6/site-packages
Finished processing dependencies for st2common==3.4.dev0
+ [[ -f /home/circleci/repo/requirements.txt ]]
+ echo 'Installing pack requirements from /home/circleci/repo/requirements.txt'
Installing pack requirements from /home/circleci/repo/requirements.txt
+ /home/circleci/virtualenv/bin/pip install -r /home/circleci/repo/requirements.txt
Collecting git+https://github.com/ibm-watson-iot/maximo-asset-monitor-sdk.git (from -r /home/circleci/repo/requirements.txt (line 2))
  Cloning https://github.com/ibm-watson-iot/maximo-asset-monitor-sdk.git to /tmp/pip-o1f94hft-build
Warning: Permanently added the RSA host key for IP address '140.82.113.3' to the list of known hosts.
Collecting pandas-schema (from -r /home/circleci/repo/requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/9c/03/6d87ce8719dc57e44688096c05fb0efa61a08c6838816c9d991b1ece5b24/pandas_schema-0.3.5-py3-none-any.whl
Collecting jsonschema>=3.2.0 (from mam-sdk==0.0.0->-r /home/circleci/repo/requirements.txt (line 2))
  Cache entry deserialization failed, entry ignored
  Downloading https://files.pythonhosted.org/packages/c5/8f/51e89ce52a085483359217bc72cdbf6e75ee595d5b1d4b5ade40c7e018b8/jsonschema-3.2.0-py2.py3-none-any.whl (56kB)
    100% |████████████████████████████████| 61kB 6.6MB/s eta 0:00:01
Collecting iotfunctions@ git+https://github.com/ibm-watson-iot/functions.git@production#egg=iotfunctions (from mam-sdk==0.0.0->-r /home/circleci/repo/requirements.txt (line 2))
  Could not find a version that satisfies the requirement iotfunctions@ git+https://github.com/ibm-watson-iot/functions.git@production#egg=iotfunctions (from mam-sdk==0.0.0->-r /home/circleci/repo/requirements.txt (line 2)) (from versions: )
No matching distribution found for iotfunctions@ git+https://github.com/ibm-watson-iot/functions.git@production#egg=iotfunctions (from mam-sdk==0.0.0->-r /home/circleci/repo/requirements.txt (line 2))
You are using pip version 9.0.3, however version 21.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

Exited with code exit status 1
CircleCI received exit code 1`
**************************************************************************

We had removed the pip restriction in the package "git@github.com:Anshika-Gautam/ci.git ~/ci" and after that we are able to download all the dependencies for our pack. But the run test step is failing for the pack now as shown in snapshot below:
<img width="910" alt="Screenshot 2021-02-19 at 7 18 13 PM" src="https://user-images.githubusercontent.com/78909008/108514027-58576100-72e9-11eb-9946-79dcdf0425e2.png">